style.css

```diff
@@ -84,3 +125,221 @@ img {
   column-gap: 20px;
   align-content: space-between;
 }
+
+.work-pages {
+  display: grid;
+  grid-template-columns: auto;
+  grid-template-rows: auto auto auto auto auto auto auto;
+  align-items: center;
+  margin-left: 24px;
+  margin-right: 24px;
+  gap: 15px;
+}
+
+.work-pages main {
+  margin-bottom: 24px;
+  display: flex;
+  flex-direction: column;
+}
+
+.title {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 52px;
+  color: #1c1a19;
+  display: flex;
+  align-items: center;
+  padding-top: 120px;
+}
+
+.line {
+  height: 0;
+  width: 427px;
+  display: flex;
+  margin-top: 50px;
+  margin-bottom: 100px;
+  border: 0.02px solid #212121;
+}
+
+.work-pages .picture {
+  background-image: url(images/Img\ Placeholder.png);
+  background-repeat: no-repeat;
+  object-fit: contain;
+  background-size: 27.5rem;
+  width: 30rem;
+  height: 20rem;
+}
+
+.work-pages main div h3 {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+  padding-top: 20px;
+}
+
+.work-pages main div p {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  padding-top: 7px;
+}
+
+.list-tag {
+  list-style-type: none;
+  display: grid;
+  grid-template-columns: auto auto auto auto;
+  gap: 20px;
+  grid-row: 1;
+  padding-top: 20px;
+  padding-bottom: 15px;
+}
+
+.tags {
+  border: 1px solid #8993a4;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  color: #091e42;
+  text-align: center;
+  padding: 10px 12px;
+}
+
+.buttons {
+  background: #ff6b00;
+  padding: 12px;
+  gap: 10px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  letter-spacing: 0.03em;
+  border-width: 0;
+}
+
+.buttons:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+.buttons:active {
+  background: #e05304;
+  color: #f4f1f0;
+}
+
+.buttons:disabled {
+  background: #f4f1f0;
+  color: #5b5957;
+}
+
+.first-page {
+  width: 95%;
+}
+
+.first-page button {
+  width: 170px;
+}
+
+.other-pages {
+  list-style-type: none;
+}
+
+.profession {
+  list-style-type: none;
+  background-image: url(images/work\ background.png);
+  width: 96%;
+}
+
+.profession .list-tag {
+  margin-left: 24px;
+}
+
+.profession .tags {
+  color: #fff;
+}
+
+.profession button {
+  width: 435px;
+}
```

Comment:
- [ Optional] You did great work however, in order for your Portfolio to look professional it will be better to change the button's width to match the Figma design. I recommend using `% ` instead of `px`, this way whatever the page width is, the button proportion will change accordingly.
- So instead of `width: 435px`  I recommend `width: 100%`